ts-sdk/whirlpool/src/createPool.ts

```diff
@@ -56,6 +95,40 @@ export function createSplashPoolInstructions(
   );
 }
 
+/**
+ * Creates the necessary instructions to initialize a Concentrated Liquidity Pool (CLMM) on Orca Whirlpools.
+ *
+ * @param {Rpc<GetMultipleAccountsApi & GetMinimumBalanceForRentExemptionApi>} rpc - A Solana RPC client for communicating with the blockchain.
```

Comment:
`Rpc<GetMultipleAccountsApi & GetMinimumBalanceForRentExemptionApi>` this is quite long and not really meaningful. Potentially we can just change it to `SolanaRpc` for the docs? `SolanaRpc` contains all the solana rpc methods so also the two we specified here.